python/rikai/viz.py

```diff
@@ -43,23 +43,26 @@ class Style(Drawable):
 
     def __init__(self, **kwarg):
         self.kwargs = kwarg
-        self.inner = None  # type: Optional[Drawable]
+        self.inner = None  # type: Optional[list[Drawable]]
 
     def __repr__(self):
         return f"style({self.kwargs})"
 
-    def __call__(self, inner: Drawable) -> Drawable:
+    def __call__(self, inner: Union[Drawable, list[Drawable]]) -> Drawable:
         # Make a copy of Style so the same style can be applied
         # to multiple drawables
         s = Style(**self.kwargs)
+        if isinstance(inner, Drawable):
+            inner = [inner]
         s.inner = inner
         return s
 
     def _render(self, render: Renderer, **kwargs):
         assert self.inner is not None
```

Comment:
forgot this in the last PR but asserts are removed in "production"

Reply:
ok, will change this to rase ValueError then